code/modules/paperwork/paper.dm

```diff
@@ -88,6 +88,7 @@
 	return
 
 /obj/item/weapon/paper/attack_ai(var/mob/living/silicon/ai/user as mob)
+	user.delayNextAttack(10)
```

Comment:
Will this really work?
I think you should add this delay to using papers on cameras instead, because this just looks like it'll delay the AI with user.